packages/block-library/src/html/edit.js

```diff
@@ -20,7 +20,22 @@ class HTMLEdit extends Component {
 
 	componentDidMount() {
 		const { styles } = this.props;
-		this.setState( { styles: transformStyles( styles ) } );
+
+		// Default styles used to unset some of the styles
+		// that might be inherited from the editor style.
+		const defaultStyles = `
+			html,body,:root {
+				margin: 0 !important;
+				padding: 0 !important;
+				overflow: visible !important;
+				min-height: auto !important;
```

Comment:
Did the overflow and min-height solve other issues, or are they pre-emptive?

Reply:
Yeah good question. 

The two questionable rules are overflow and min-height, and I _believe_ I added those because many themes use overflow as an "auto clearing" measure, and min-height for sites that need to leverage the full viewport height above the fold. 

I can't recall why I unstyled those for the block styles preview but I know they were necessary. 

@kjellr sorry for pinging you the third time today, you're my CSS Yoda — what side effects might there be to changing the overflow and min-height properties? I'm leaning towards these being fine for the purposes of teh HTML preview tab, but would appreciate your thoughts.

Reply:
>  what side effects might there be to changing the overflow and min-height properties? I'm leaning towards these being fine for the purposes of teh HTML preview tab, but would appreciate your thoughts.

I did some general testing, and the styles didn't seem to cause any issues either way (keeping these `!important` styles, or removing them). I don't necessarily think it'll be a problem to include them, but I can't figure out what problem they're solving.

Reply:
The problem they may solve is if the editor styles include things like `html body` or something, which is definitely very rare but I just thought about being certain as the intent is to override the styles no matter what.

Reply:
Ah, I see. Yeah, it probably makes sense to include these as a preventative measure then.